homeassistant/components/influxdb/sensor.py

```diff
@@ -173,13 +227,28 @@ def setup_platform(
                 entities.append(InfluxSensor(hass, influx, query))
             else:
                 _LOGGER.error(NO_BUCKET_ERROR, query[CONF_BUCKET])
-    else:
+
+    if CONF_QUERIES in config:
         for query in config[CONF_QUERIES]:
             if query[CONF_DB_NAME] in influx.data_repositories:
                 entities.append(InfluxSensor(hass, influx, query))
             else:
                 _LOGGER.error(NO_DATABASE_ERROR, query[CONF_DB_NAME])
 
+    if CONF_QUERIES_RAW in config:
+        for query in config[CONF_QUERIES_RAW]:
+            if query[CONF_API_VERSION] == DEFAULT_API_VERSION:
+                if query[CONF_DB_NAME] in influx.data_repositories:
+                    entities.append(InfluxSensor(hass, influx, query))
+                else:
+                    _LOGGER.error(NO_DATABASE_ERROR, query[CONF_DB_NAME])
+
+            if query[CONF_API_VERSION] == API_VERSION_2:
+            if query[CONF_BUCKET] in influx.data_repositories:
+                entities.append(InfluxSensor(hass, influx, query))
+            else:
+                _LOGGER.error(NO_BUCKET_ERROR, query[CONF_BUCKET])
```

Comment:
Indentation issue, just indent all this one level more.

Reply:
Yes, sorry not to havec done it before... I commit a new version with the correction.